readthedocs/api/v2/views/integrations.py

```diff
@@ -367,21 +367,22 @@ def handle_webhook(self):
         if event in (GITHUB_CREATE, GITHUB_DELETE):
             return self.sync_versions(self.project)
 
-        if event == GITHUB_PULL_REQUEST and action:
-            if (
-                action in
-                [
-                    GITHUB_PULL_REQUEST_OPENED,
-                    GITHUB_PULL_REQUEST_REOPENED,
-                    GITHUB_PULL_REQUEST_SYNC
-                ]
-            ):
-                # Handle opened, synchronize, reopened pull_request event.
-                return self.get_external_version_response(self.project)
-
-            if action == GITHUB_PULL_REQUEST_CLOSED:
-                # Handle closed pull_request event.
-                return self.get_delete_external_version_response(self.project)
+        if self.project.has_feature(Feature.ENABLE_EXTERNAL_VERSION_BUILD):
+            if event == GITHUB_PULL_REQUEST and action:
```

Comment:
these can be joined together with an `and`, instead of a nested `if`.

Reply:
Done! :)